packages/core/src/RPA/core/locators.py

```diff
@@ -34,6 +35,31 @@ def open_stream(obj, *args, **kwargs):
             obj.close()
 
 
+def find_by_name_or_error(path, criteria):
```

Comment:
maybe just `find_by_name` would be enough

Reply:
Well the LocatorDatabase exposes a `find_by_name` method that doesn't fit the need of `Browser` module or the needs image aliases will soon have.

Maybe `load_by_name` could work, since it would represent that we actually load the db from disk when this function is called?

Reply:
That would work, I think. Some other "loading" can also happen behind the scenes, like parsing images

Reply:
Changed at https://github.com/robocorp/rpaframework/pull/58/files